src/patterns/ec2-app.test.ts

```diff
@@ -122,24 +136,36 @@ describe("the GuEC2App pattern", function () {
       userData: "",
     });
 
-    expect(stack).toHaveResource("AWS::EC2::SecurityGroup", {
-      SecurityGroupIngress: [
-        {
-          CidrIp: "192.168.1.1/32",
-          Description: "Allow access on port 443 from 192.168.1.1/32",
-          FromPort: 443,
-          IpProtocol: "tcp",
-          ToPort: 443,
-        },
-        {
-          CidrIp: "8.8.8.8/32",
-          Description: "Allow access on port 443 from 8.8.8.8/32",
-          FromPort: 443,
-          IpProtocol: "tcp",
-          ToPort: 443,
-        },
-      ],
-    });
+    const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
+    const loadBalancerKey = Object.keys(json.Resources).filter(
+      (resource) => json.Resources[resource].Type === "AWS::ElasticLoadBalancingV2::LoadBalancer"
+    )[0];
+    const loadBalancer = json.Resources[loadBalancerKey];
+    const lbSecurityGroups = loadBalancer.Properties.SecurityGroups as [Record<string, never>];
+    const lbSecurityGroupID = lbSecurityGroups[0]["Fn::GetAtt"][0] as string;
+    const lbSecurityGroup = json.Resources[lbSecurityGroupID];
```

Comment:
This is a bit of a hacky way to ensure the specific security group for the load balancer is the one we're asserting over, as the previous test failed to assert that the _specific_ security group was the one with the ingress rules.